concepticondata/conceptlists/Pagel-2018-200.tsv

```diff
@@ -87,10 +87,10 @@ Pagel-2018-200-85	85	laugh	1355	LAUGH	0.00012791
 Pagel-2018-200-86	86	leaf	628	LEAF	0.000226195
 Pagel-2018-200-87	87	left	244	LEFT	0.000291936
 Pagel-2018-200-88	88	leg	1297	LEG	0.000293917
-Pagel-2018-200-89	89	lie			0.000226318
+Pagel-2018-200-89	89	lie	215	LIE DOWN	0.000226318
 Pagel-2018-200-90	90	live	1422	BE ALIVE	7.1041E-05
 Pagel-2018-200-91	91	liver	1224	LIVER	0.000168724
-Pagel-2018-200-92	92	long			0.000127153
+Pagel-2018-200-92	92	long	2143	LONG OR TALL	0.000127153
```

Comment:
sorry, but long is still 1203 long, not long or tall (IELex and ABVD both link to LONG).

Reply:
You don't have to be sorry, I'm a little bit confused and not paying the attention I should be paying. Sorry about that.